ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -44,9 +49,36 @@ export default class StatusDropdown extends React.Component {
       showNotesModal: false,
       showTagsModal: false,
       selectedValue: this.props.issueTrackers[0].text,
+      jobList: [],
     };
   }
 
+  findRelevantAlert = () => {
+    const { alertSummary } = this.props;
+    let relevantAlert = alertSummary.alerts.find((alert) => alert.status === 0);
+    // eslint-disable-next-line prefer-destructuring
+    if (relevantAlert.length > 1) relevantAlert = relevantAlert[0];
+    return relevantAlert;
+  };
+
+  // eslint-disable-next-line camelcase
+  getTaskId = async (push_id) => {
+    const jobList = await JobModel.getList({ push_id });
+    this.setState({ jobList });
+    const relevantAlert = this.findRelevantAlert();
+
+    const relevantJob = jobList.data.find(
+      (job) =>
+        job.searchStr.includes(relevantAlert.series_signature.suite) &&
+        job.searchStr.includes(relevantAlert.series_signature.machine_platform),
```

Comment:
We should filter by framework (it's possible for the same suite to be available on multiple frameworks).

Reply:
the framework condition is at line 144. The added functions will be called only if the framework is `browsertime`